src/components/ChannelList/__tests__/ChannelList.test.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable sonarjs/no-unused-collection */
+import React from 'react';
+import { cleanup, render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { Chat } from '../../Chat';
+import ChannelList from '../ChannelList';
+import {
+  getTestClient,
+  useMockedApis,
+  queryChannelsApi,
+  generateMessage,
+  generateChannel,
+  generateUser,
+  getOrCreateChannelApi,
+  dispatchMessageNewEvent,
+  dispatchNotificationAddedToChannelEvent,
+} from '../../../__mocks__';
```

Comment:
@jaapbakker88 @vishalnarkhede we can setup babel module resolver to get rid of all these `../../..`, at least for tests